build_script.sh

```diff
@@ -9,4 +18,17 @@ fi
 
 cd build
 cmake ..
-cmake --build . -j$CPUS
\ No newline at end of file
+cmake --build . -j$CPUS
+
+cd "${topDir}"
+cp -f build/src/pythonmonkey.so python/pythonmonkey/
+(
+  echo "# This file was generated via $0 by `id -un` on `hostname` at `date` - do not edit by hand!"
+  grep '^version' pyproject.toml \
+  | head -1 \
+  | sed 's/^version /__version__ /' \
+) > python/pythonmonkey/version.py
```

Comment:
This should be managed by [poetry-dynamic-versioning](https://github.com/mtkennerly/poetry-dynamic-versioning) instead
https://github.com/Distributive-Network/PythonMonkey/blob/43d610d/pyproject.toml#L36-L40

Reply:
We already have poetry-dynamic-versioning in `main`